contribs/dvrp/src/main/java/org/matsim/contrib/dvrp/passenger/PassengerRequestSubmittedEvent.java

```diff
@@ -41,9 +45,9 @@ public class PassengerRequestSubmittedEvent extends AbstractPassengerRequestEven
 	private final Id<Link> fromLinkId;
 	private final Id<Link> toLinkId;
 
-	public PassengerRequestSubmittedEvent(double time, String mode, Id<Request> requestId, Id<Person> personId,
+	public PassengerRequestSubmittedEvent(double time, String mode, Id<Request> requestId, Set<Id<Person>> personIds,
```

Comment:
Please change from `Set` to `List` in all events. `List` is better for events (computations + memory).

BTW. Using a list does not mean that we cannot guarantee uniqueness of elements.